google-cloud-examples/src/main/java/com/google/cloud/examples/storage/snippets/StorageSnippets.java

```diff
@@ -1393,4 +1526,74 @@ public Blob releaseTemporaryHold(String bucketName, String blobName) throws Stor
     // [END storage_release_temporary_hold]
     return blob;
   }
+
+  /**
+   * Example of how to enable Bucket Policy Only for a bucket
+   */
+  public Bucket enableBucketPolicyOnly(String bucketName) throws StorageException {
+    // [START storage_enable_bucket_policy_only]
+    // Instantiate a Google Cloud Storage client
+    Storage storage = StorageOptions.getDefaultInstance().getService();
+
+    // The name of a bucket, e.g. "my-bucket"
+    // String bucketName = "my-bucket";
```

Comment:
I think this should be uncommented?

Reply:
Oh nvm, I see it's being passed through as a parameter. Can we remove the passed in parameter and leave the code above for clarity?

Reply:
@kolea2, thanks for the feedback.

The passed in parameter is used by the integration test and the commented variable could be copy/pasted by a developer into their code. I'm using existing sample design followed by this file. 

Could you add an example of what you'd like to see?

Reply:
Ah, ok makes sense - didn't realize the parameters were used in that way. That seems like the best approach then!